### DISCUSSION TITLE
Why isn't NetPanzer on Steam and will it be

### DISCUSSION THREAD

**Discussion by andy5995** [netpanzer/netpanzer#101] [General]
Some good reasons to wait before putting it on Steam. A discussion about [0ad](https://play0ad.com/) on Steam is at https://wildfiregames.com/forum/topic/20461-0-ad-on-steam/#comment-313456 and https://wildfiregames.com/forum/topic/95468-why-not-moving-this-to-steam/#comment-523815 There are some valid reasons to wait for more stability and testing before it's on Steam.

One big reason is that any undiscovered bugs may lead to bad reviews. And the reviews stay on there permanently.

I believe we can increase the popularity of NetPanzer without Steam. Once it's released, most Linux distros will update. And we're going to make it more accessible by having an AppImage for Linux. Eventually someone will do a flatpack and snap package.

For Windows, we'll be making the binary very available.

But feel free to discuss further. I don't make the final decisions around here :)

**Comment by andy5995** [comment]
There's lots of different, free ways, to promote NP. It does take time, but a little bit every week accumulates over time.

We could also consider having a promotion discussion thread as well to offer ideas and link to where we've mentioned NetPanzer, or where it's already mentioned.

**Comment by winrid** [comment]
Well, you can do pre-release on Steam to manage expectations, so I think the bugs part is not so bad.

The main issue is the GPL.

Anyways, we will solve the auto-update issue with #35 and we can have a donate button on the site that goes to a Netpanzer account/llc (maybe).

Plenty of popular games are not on Steam.
